src/components/CodeiumEditor/CompletionProvider.ts

```diff
@@ -64,11 +64,11 @@ export class MonacoCompletionProvider {
 
   ) {
     this.metadata = new Metadata({
-      ideName: getBrowserVersion() ?? "unknown",
-      ideVersion: getCurrentURL() ?? "unknown",
```

Comment:
Also one reason why this might be undefined is that it's being cached on init. One thing I was thinking was that on NextJS apps, it'll be undefined because of SSR. So if we refresh this value when a request is sent that'd probably fix the issue

Reply:
Hopefully fixed by #23